### DISCUSSION TITLE
听歌系列之罗文《黄昏》 | /cn/2019/06/dusk/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1335] [Comments]
https://yihui.org/cn/2019/06/dusk/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
本乐盲正好借问一下，前奏中悠扬的究竟是啥乐器啊？

> ——原帖发布于 2019-07-25 21:21:22

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
这本书超长，而且脚注极其多。我已经读了四五个月了，终于快读完了。

> ——原帖发布于 2019-10-24 14:35:53

[a comment by an automated account is not shown]

**Comment by xiaomeng3** [comment]
写得真好，期待博主再更。

罗文《黄昏》的官方作词、作曲人说是夏浩杰，但此人隐姓埋名、履历未知。

**Reply by yihui** [reply]
更文不知何时，歌倒是可以先更一首：https://kg.qq.com/node/play?s=0DWPWlB0KEzNGB9z& 近日大咳一场尚未完全复原，高音有些吃力，局部地区有哑嗓现象。
